applications/asset_tracker/src/main.c

```diff
@@ -411,12 +413,14 @@ static void modem_rsrp_data_send(struct k_work *work)
 	sensor_data_send(&signal_strength_cloud_data);
 	timestamp_prev = k_uptime_get_32();
 }
+#endif
```

Comment:
Could you add a comment for what the closing `endif` is for? Like `#endif /* CONFIG_MODEM_INFO */`